pandas/indexes/multi.py

```diff
@@ -5,6 +5,11 @@
 from functools import partial
 from sys import getsizeof
 
+try:
+    from itertools import zip_longest
+except ImportError:
```

Comment:
this should be in pandas.compat (might be there already actually) so put / import from there